server/src/test/java/io/crate/expression/scalar/arithmetic/SquareRootFunctionTest.java

```diff
@@ -39,12 +41,15 @@ public void testSqrt() {
         assertEvaluateNull("sqrt(null)");
         assertEvaluate("sqrt(cast(25 as integer))", 5.0);
         assertEvaluate("sqrt(cast(25.0 as float))", 5.0);
+        assertEvaluate("sqrt(cast(225.0 as numeric(6, 1)))", new BigDecimal("15.0"));
```

Comment:
can we add an example where answer is not integer, so demostrate that argument precision/scale is applied?

Reply:
done